From 89be40e1364676128830ffbc8ca668659ec48ba2 Mon Sep 17 00:00:00 2001
From: Dennis Bonke <admin@dennisbonke.com>
Date: Tue, 27 Jan 2026 21:36:09 +0100
Subject: [PATCH] Don't build tests as they can't be cross compiled

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 5985ab6..3261c22 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -36,7 +36,7 @@ if ENABLE_DOC
 SUBDIRS += doc examples
 endif
 
-SUBDIRS += fuzz tests
+SUBDIRS += fuzz
 
 ABIDW_COMMON = --no-show-locs --no-corpus-path
 ABIGNORE_FILE = "$(top_srcdir)/devel/libtasn1.abignore"
-- 
2.51.0

