From 0749e24c8bec5388139aae2e3901b42d3ef6c3e4 Mon Sep 17 00:00:00 2001
From: czapek1337 <czapek1337@gmail.com>
Date: Mon, 9 May 2022 16:44:45 +0200
Subject: [PATCH] Accept linux-mlibc as a valid OS

---
 config.sub     | 4 ++--
 test/picsmap.c | 2 ++
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/config.sub b/config.sub
index 0f2234c..a09757d 100755
--- a/config.sub
+++ b/config.sub
@@ -131,7 +131,7 @@ case $1 in
 		# parts
 		maybe_os=$field2-$field3
 		case $maybe_os in
-			nto-qnx* | linux-gnu* | linux-android* | linux-dietlibc \
+			nto-qnx* | linux-gnu* | linux-android* | linux-dietlibc | linux-mlibc* \
 			| linux-newlib* | linux-musl* | linux-uclibc* | uclinux-uclibc* \
 			| uclinux-gnu* | kfreebsd*-gnu* | knetbsd*-gnu* | netbsd*-gnu* \
 			| netbsd*-eabi* | kopensolaris*-gnu* | cloudabi*-eabi* \
@@ -1355,7 +1355,7 @@ case $os in
 	     | chorusrdb* | cegcc* | glidix* \
 	     | cygwin* | msys* | pe* | moss* | proelf* | rtems* \
 	     | midipix* | mingw32* | mingw64* | linux-gnu* | linux-android* \
-	     | linux-newlib* | linux-musl* | linux-uclibc* \
+	     | linux-newlib* | linux-musl* | linux-uclibc* | linux-mlibc* \
 	     | uxpv* | beos* | mpeix* | udk* | moxiebox* \
 	     | interix* | uwin* | mks* | rhapsody* | darwin* \
 	     | openstep* | oskit* | conix* | pw32* | nonstopux* \
diff --git a/test/picsmap.c b/test/picsmap.c
index 432fbf7..b782ad4 100644
--- a/test/picsmap.c
+++ b/test/picsmap.c
@@ -53,6 +53,8 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
+#include <strings.h>
+
 #if HAVE_STDINT_H
 #include <stdint.h>
 #define my_intptr_t	intptr_t
-- 
2.51.0

